yarn-project/circuits.js/src/contract/contract_instance_deployed_event.ts

```diff
@@ -34,7 +34,7 @@ export class ContractInstanceDeployedEvent {
     const salt = reader.readObject(Fr);
     const contractClassId = reader.readObject(Fr);
     const initializationHash = reader.readObject(Fr);
-    const portalContractAddress = reader.readObject(EthAddress);
+    const portalContractAddress = EthAddress.fromField(reader.readObject(Fr));
```

Comment:
https://github.com/AztecProtocol/aztec-packages/blob/42c9124d7b049a8c03695f1df0b88d4b72ad256c/yarn-project/circuit-types/src/l1_to_l2_message.ts#L190

based on this line by did this need to change?

Reply:
The number of bytes read (ie how many bytes the buffer reader advances) changed from 32 to 20 in #4442.